README.md

```diff
@@ -55,7 +55,7 @@ A full example leveraging other community modules is contained in the [test/fixt
 2. Define the ALB module using the following in your [main.tf](test/fixtures/main.tf):
 ```
 module "alb" {
-  source              = "github.com/terraform-community-modules/tf_aws_alb//alb"
+  source              = "github.com/terraform-community-modules/tf_aws_alb/alb"
```

Comment:
There should be double slashes.

`Note: The double-slash, //, is important. It is what tells Terraform that that is the separator for a subdirectory, and not part of the repository itself.` from https://www.terraform.io/docs/modules/sources.html